### PR TITLE
Add bulk assess view, refactor verify assessment data

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Add a bulk staff assess handler to staff_assessment_mixin

JIRA: [AU-51](https://openedx.atlassian.net/browse/AU-51)

**What changed?**

- Refactor verify_assessment_parameters logic out to a separate method
- Wrote a new verify_multiple_assessment_parameters to handle a list of assessments rather than a single assessment
- Refactored almost all of staff_assess handler out to a helper, _do_staff_assess
- wrote bulk_staff_assess which calls _do_staff_assess for each staff assessment received. 
- tests

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- Make an ORA
- Submit at least two submissions for the ORA
- follow instructions in https://openedx.atlassian.net/wiki/spaces/ENG/pages/3145892500/How+to+authenticate+and+query+edX+APIs+with+Postman to mke a POST as a staff userto Request URL: `https://localhost:18000/courses/<course_id>/xblock/<block _id>/handler/bulk_staff_assess`. Parameter info is included in the view docstring.
- You should receive a response, and as long as there were not any errors, the submissions should be assessed.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/test/test_staff.py [hunk 1]
```diff
@@ -268,6 +269,234 @@ def test_assessment_error(self, xblock):
...
+        derek_good_assessment = self._build_assessment_dict('Derek', test_submissions)
+
+        # Expect the response to fail and for the response to include error info
+        payload = [
+            alice_no_options_selected, billy_no_criterion_feedback, cindy_no_overall_feedback, derek_good_assessment
+        ]
+        resp = self.request(xblock, 'bulk_staff_assess', json.dumps(payload), response_format='json')
+        self.assertFalse(resp['success'])
+        self.assertEqual(resp['msg'], "One or more of the submitted assessments is missing required fields")
+        breakpoint()
```
Comment (mattcarter) [inline openassessment/xblock/test/test_staff.py:425]: Looks like that needs to come out :)

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Review by nsprenkle — COMMENTED** [review]
Overall, looks good. A note on organization: I wonder if we want to isolate Staff Grader functions/helpers in their own folder/location.

**Review by nsprenkle — APPROVED** [review]
Vote for approve/merge and reorganize later :shipit:
